### PR TITLE
reference latest version 0.3.4 in getting started guide

### PR DESCRIPTION
**Pull request by bwagner5**
**1. Issue, if available:**
N/A

**2. Description of changes:**
 - Update getting started guide to reference latest version 0.3.4

**3. Does this change impact docs?**
- [x] Yes, PR includes docs updates
- [ ] Yes, issue opened: *link to issue*
- [ ] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/docs/getting-started/_index.md [hunk 1]
```diff
@@ -177,7 +177,13 @@ metadata:
 spec:
   provider:
     instanceProfile: KarpenterNodeInstanceProfile-${CLUSTER_NAME}
-    capacityType: spot
+    capacityTypes: [spot]
+    architectures: [ "amd64" ]
```
Comment (ellistarn) [inline website/content/en/docs/getting-started/_index.md:181]: I'd leave this out.

File: website/content/en/docs/getting-started/_index.md [hunk 2]
```diff
@@ -177,7 +177,13 @@ metadata:
 spec:
   provider:
     instanceProfile: KarpenterNodeInstanceProfile-${CLUSTER_NAME}
-    capacityType: spot
+    capacityTypes: [spot]
```
Comment (ellistarn) [inline website/content/en/docs/getting-started/_index.md:180]: I don't think this is valid YAML. You need `"strings"` unless you're using `-` syntax

File: website/content/en/docs/getting-started/_index.md [hunk 3]
```diff
@@ -177,7 +177,13 @@ metadata:
...
+    architectures: [ "amd64" ]
+    operatingSystems: [ "linux" ]
```
Comment (ellistarn) [inline website/content/en/docs/getting-started/_index.md:182]: I'd leave this out, too.

File: website/content/en/docs/getting-started/_index.md [hunk 4]
```diff
@@ -177,7 +177,13 @@ metadata:
...
+    subnetSelector:
+      kubernetes.io/cluster/${CLUSTER_NAME}: "*"
+    securityGroupSelector:
+      eksctl.cluster.k8s.io/v1alpha1/cluster-name: "${CLUSTER_NAME}"
```
Comment (ellistarn) [inline website/content/en/docs/getting-started/_index.md:186]: Why not just allow these both to default?

File: website/content/en/docs/getting-started/_index.md (hunk 2 above)
Comment (bwagner5) [reply website/content/en/docs/getting-started/_index.md:180]: works for me, but I'll make it uniform to match the rest.

**Review by ellistarn — APPROVED** [review]

**Pull request merged**